extensions/tasks-parent/tasks-client/src/test/java/io/resys/thena/tasks/tests/TaskHistoryTest.java

```diff
@@ -0,0 +1,103 @@
+package io.resys.thena.tasks.tests;
+
+/*-
+ * #%L
+ * thena-tasks-client
+ * %%
+ * Copyright (C) 2021 - 2023 Copyright 2021 ReSys OÜ
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.resys.thena.tasks.client.api.model.ImmutableAssignTaskReporter;
+import io.resys.thena.tasks.client.api.model.ImmutableChangeTaskPriority;
+import io.resys.thena.tasks.client.api.model.ImmutableChangeTaskStatus;
+import io.resys.thena.tasks.client.api.model.ImmutableCreateTask;
+import io.resys.thena.tasks.client.api.model.Task;
+import io.resys.thena.tasks.tests.config.TaskPgProfile;
+import io.resys.thena.tasks.tests.config.TaskTestCase;
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONException;
+import org.junit.jupiter.api.Assertions;
+
+import java.time.Duration;
+
+@Slf4j
+@QuarkusTest
+@TestProfile(TaskPgProfile.class)
+public class TaskHistoryTest extends TaskTestCase {
+
+  private static final String repoName = TaskUpdateTest.class.getSimpleName();
```

Comment:
The class name is incorrect here.   It might lead to interesting errors someday...(aka test functions on one machine but don't on another).